src/common/types/store-data/unified-data-interface.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// this is similar to `TestEngine` interface from axe-core
+export interface ScanEngine {
+    name: string;
+    version: string;
+}
+
+export interface OSData {
+    name: string;
+    osVersion?: string;
```

Comment:
The interface already has "Os" in it; so "os" in "osVersion" is redundant.

Reply:
Thanks for the feedback on "OS" :) I will just remove that prefix from "version"